src/data/mainnet/celo-tokens-info.json

```diff
@@ -653,7 +654,10 @@
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/eXOF.png",
     "isCoreToken": true,
     "name": "ECO CFA",
-    "symbol": "eXOF"
+    "symbol": "eXOF",
+    "isStableCoin": true,
+    "isCashOutEligible": true,
```

Comment:
Can you remove these `isCashOutEligible` fields? We actually need some work in the wallet code first to be able to support a new cash out currency. 

Btw, what cash out providers support eXOF?